src/components/dropdown-filter/dropdown-filter.scss

```diff
@@ -22,13 +24,26 @@
   box-sizing: border-box;
   cursor: pointer;
   left: 0;
-  padding: 5px 6px;
+  padding: 6px 6px 5px 6px;
   position: absolute;
   width: 100%;
   z-index: $z-dropdown-list;
 
   &:hover {
-    background-color: $blue-bright;
-    color: $white;
+    background-color: #E6E9E7;
+  }
+}
+
+.carbon-dropdown-filter {
+  .carbon-link__content {
```

Comment:
should this use the `.carbon-dropdown__action` namespace instead?

This could target _any_ link inside of dropdown filter, whereas `__action` will target this specific link.